app/code/community/Sailthru/Email/etc/config.xml

```diff
@@ -9,7 +9,7 @@
 <config>
     <modules>
         <Sailthru_Email>
-            <version>2.1.0</version>
+            <version>2.1.0-beta</version>
```

Comment:
Would this be the release version displayed? And will you update this to remove beta when merged into master?